web/src/recent_view_ui.ts

```diff
@@ -357,8 +376,10 @@ export function revive_current_focus() {
             // If the only message in the topic was deleted,
             // then the topic will not be in Recent Conversations data.
             if (recent_view_data.conversations.get(last_visited_topic) !== undefined) {
-                const topic_last_msg_id =
-                    recent_view_data.conversations.get(last_visited_topic).last_msg_id;
+                const last_conversation = recent_view_data.conversations.get(last_visited_topic);
+                assert(last_conversation !== undefined);
+                const topic_last_msg_id = last_conversation.last_msg_id;
+                assert(topics_widget !== undefined);
```

Comment:
This assert probably most correctly belongs on the line just after `is_table_focused()` -- since that's the check that makes us know this is set.